docs/release-notes.txt

```diff
@@ -79,6 +79,7 @@ This release includes the following new features:
 - When 0 is given as the max connection pool size, it is now interpreted to
   mean no limit.
 - The default maximum connection pool size has been increased to 20 from 5.
+- Added support for enumerables in bulk write operations.
```

Comment:
Why won't it need a release not even if we're backporting to 2.18?

Reply:
Because the break was in 2.18, thus after the fix there wouldn't be any stable branch where the issue exists so it might be confusing to people to try to work out which versions are broken.

But if you wish to have the note, that's fine, I would just phrase it as "Reinstated support for enumerables" or similar since it was already present and unintentionally broken.

Reply:
oh okay I will remove it